components/helper/ConfirmModal.tsx

```diff
@@ -0,0 +1,83 @@
+import { Modal, Box, Typography, Button, createTheme, ThemeProvider }
+  from '@mui/material';
+import { grey } from '@mui/material/colors';
+
+function ConfirmModal({
+  open,
+  onClose,
+  message,
+  onConfirm,
+  onCancel,
+  confirmButtonName,
+  colorButton
+}) {
+
+  const theme = createTheme({
+    components: {
+      MuiButton: {
+        styleOverrides: {
+          root: {
+            fontWeight: 'bold',
+            letterSpacing: 'normal',
+            textTransform: 'none',
+          },
+        },
+        variants: [
+          {
+            props: { variant: 'contained', color: 'primary' },
+            style: {
+              marginLeft: '0',
+              backgroundColor: colorButton[500],
+              '&:hover': {
+                backgroundColor: colorButton[700],
+              },
+              '&:active': {
+                backgroundColor: colorButton[300],
+              },
+            },
+          },
+          {
+            props: { variant: 'contained', color: 'secondary' },
+            style: {
+              marginLeft: '1rem',
+              backgroundColor: grey[500],
+              '&:hover': {
+                backgroundColor: grey[700],
+              },
+              '&:active': {
+                backgroundColor: grey[300],
+              },
+            },
+          },
+        ],
+      },
+    },
+  })
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Modal open={open} onClose={onClose} id='modalConfirm' >
+        <Box sx={{
+          position: 'absolute', top: '40%', left: '50%',
```

Comment:
what happens if we don't specify the attributes?

Reply:
If we do not specify the attributes, the <Box> element will not have the desired dimensions or styles, or did I not understand the question?